.github/workflows/macos.yml

```diff
@@ -60,4 +60,5 @@ jobs:
 
       - name: Run tests
         run: |
-          pytest -n 3 --maxfail 3 --durations 10 tests/unit tests/functional --ignore tests/functional/test_libraries.py
+          pytest -n 3 --maxfail 3 --durations 10 tests/unit tests/functional --ignore tests/functional/test_libraries.py \
+            --force-flaky --no-success-flaky-report
```

Comment:
Should this be `--no-flaky-report` instead of `--no-success-flaky-report`? (Or both, everywhere?)

Reply:
So it should...